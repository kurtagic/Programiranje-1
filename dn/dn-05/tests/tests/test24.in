76
7 22
7 36
7 24
7 29
7 43
7 21
7 41
7 40
7 24
7 42
7 30
7 40
7 33
7 36
7 41
7 25
7 36
7 34
7 38
7 21
7 27
7 28
7 33
7 26
7 24
7 25
7 37
7 22
7 26
7 27
7 44
7 22
7 25
7 26
7 40
7 42
7 31
7 33
7 42
7 24
7 44
7 25
7 22
7 34
7 33
7 36
7 44
7 43
7 25
7 22
7 42
7 44
7 24
7 32
7 22
7 34
7 28
7 23
7 32
7 41
7 38
7 41
7 34
7 30
7 30
7 27
7 24
7 26
7 30
7 33
7 26
7 30
7 29
7 43
7 23
7 36
