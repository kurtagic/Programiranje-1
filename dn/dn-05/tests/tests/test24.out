21: 9
22: 22
23: 32
24: 30
25: 27
26: 26
27: 24
28: 15
29: 20
30: 18
31: 15
32: 20
33: 21
34: 19
35: 16
36: 16
37: 14
38: 11
39: 9
40: 13
41: 24
42: 22
43: 19
44: 17
45: 14
